helm/hpcc/templates/_helpers.tpl

```diff
@@ -406,6 +406,12 @@ Check that the storage and spill planes for a component exist
 {{- end }}
 {{- end -}}
 
+{{/*
```

Comment:
This seems (a) a bit small to be worth having a template for and (b) to be unused

Reply:
agree, will remove.